uint32 time_stamp
int16[6] rc
uint16 flight_mode
uint16 last_flight_mode
